docs/source/index.rst

```diff
@@ -62,6 +62,19 @@ termination using the message id.
 
   abort(message_id)
 
+Abort a task using a custom abort_ttl value
+------------
+
```

Comment:
Could you add a note about why this might be required?

> By default, abort has a limited window of 90,000 milliseconds. This means a worker will skip a task only if the task was aborted up to 90 seconds ago. In case of longer delay in the task processing this value can be overridden.